draft-annevk-johannhof-httpbis-cookies.md

```diff
@@ -501,18 +501,22 @@ user agent).
 #### The Domain Attribute {#attribute-domain}
 
 The Domain attribute specifies those hosts to which the cookie will be sent.
-For example, if the value of the Domain attribute is "site.example", the user
-agent will include the cookie in the `Cookie` header field when making HTTP requests to
-site.example, www.site.example, and www.corp.site.example. (Note that a
-leading %x2E ("."), if present, is ignored even though that character is not
-permitted.)  If the server omits the Domain attribute, the user agent
-will return the cookie only to the origin server.
+
+If the server includes the Domain attribute, the value applies to both the
+specified domain and any subdomains. For example, if the value of the Domain
+attribute is "site.example", the user agent will include the cookie in the
+`Cookie` header field when making HTTP requests to site.example, www.site.example,
+and www.corp.site.example. Note that a leading %x2E ("."), if present, is
+ignored even though that character is not permitted.
+
+If the server omits the Domain attribute, the user agent will return the cookie
+only to the origin server and not to any subdomains.
```

Comment:
Is this actually accurate? Origin server seems to imply scheme and port matter, which I'm pretty sure cookies ignore. (You can make them care about scheme, but not by default.)

Might be a follow-up though.

Reply:
Perhaps replace `...only to origin server` with `...only to requests from the original domain which sent the Set-Cookie header`. But maybe that's a bit wordy.

FWIW, I didn't change the `origin server` text in that sentence - I just added the ` and not to any subdomains` bit.

Reply:
Yeah, I suspect "origin" here doesn't mean (scheme, host, port) but rather "where it came from". That seems worth fixing up, but I wouldn't put it on this PR unless @roryhewitt feels motivated.